tests/bluetooth/tester/src/bttester.h

```diff
@@ -723,6 +723,13 @@ struct gatt_change_db_cmd {
 	uint8_t visibility;
 } __packed;
 
+#define GATT_EATT_CONNECT		0x1f
+struct gatt_eatt_connect_cmd {
+	uint8_t address_type;
+	uint8_t address[6];
```

Comment:
Any reason why you're not using `bt_addr_le_t`?

Reply:
This is the way the other commands does it, so I did it the same way. Using `bt_addr_le_t` instead should work the same.